pyproject.toml

```diff
@@ -18,6 +18,7 @@ httpx-sse = {version = ">=0.3.1", optional = true}
 pydantic = ">=1"
 langchain = ">=0.0.333"
 orjson = ">=2"
+langchain-anthropic = "^0.1.4"
```

Comment:
This change means that all developers that install LangServe will be forced to install anthropic as well. Would you be able to undo the changes in the poetry and pyproject files?